### ISSUE TITLE
Need to add way for new devmode plugin to communicate with connect-phonegap

### ISSUE THREAD

**Issue by timkim** [phonegap/connect-phonegap#184]
When the new dev mode plugin connects to `connect-phonegap`, it doesn't require the injected scripts. Therefore, we need to add a way for the dev mode plugin to easily disable that feature.

**Comment by filmaj** [comment]
Is this still an issue @timkim ? I see commits referencing this have landed in master already...

**Comment by timkim** [comment]
Yes, still an issue, it is but for when we re-re-do the dev app. Basically shelved for now but should remain open.
